how-we-work/oncall-responsibilities.md

```diff
@@ -27,6 +27,7 @@ Deployments for each repository/environment are performed either automatically o
 | single-cell-infra       | Auto-deployed by TFE upon PR merge | Auto-deployed upon TFE PR merge (excl. [explorer](https://github.com/chanzuckerberg/single-cell-infra/blob/a8d1a3cc5f36280de69f7250f4a6422a55d574fc/terraform/tfe/locals.tf.json#L21) infra) | Manually deployed via TFE plan confirmation |
 | single-cell-data-portal | Auto-deployed by Github Action upon PR merge on to `main` | Manually deployed by happy/script | Manually deployed by happy/script |
 | single-cell-explorer    | Auto-deployed by Github Action upon PR merge on to `main` | Manually deployed via single-cell-infra deploy script that runs Github Action (can also run locally) | Manually deployed via single-cell-infra deploy script that runs Github Action (can also run locally) | Auto-deployed by Github Action upon PR merge (does not auto rebase off main) |
+single-cell-curation (cellxgene-schema) | n/a | [manual](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/release_process.md), if needed | manual, if needed
```

Comment:
nit: Would just point here that it's "releasing to pypi" (and not actually deploying a running a system).